Makefile.config.in

```diff
@@ -247,8 +247,6 @@ FLAMBDA=@flambda@
 WITH_FLAMBDA_INVARIANTS=@flambda_invariants@
 FORCE_INSTRUMENTED_RUNTIME=@force_instrumented_runtime@
 WITH_CMM_INVARIANTS=@cmm_invariants@
-FORCE_SAFE_STRING=@force_safe_string@
-DEFAULT_SAFE_STRING=@default_safe_string@
```

Comment:
It's very unlikely, but because we install `Makefile.config`, we usually keep the definitions themselves:
```make
FORCE_SAFE_STRING=true
DEFAULT_SAFE_STRING=true
```
(as in `utils/config.mlp`)

Reply:
Done